### DISCUSSION TITLE
Install Helper Script and Reset chicken egg problem

### DISCUSSION THREAD

**Discussion by mulder999** [Guilouz/Creality-Helper-Script-Wiki#606] [Issues]
Installer requires reset: `Please restore firmware to factory settings before using script to avoid issue.
See: Reset Factory Settings section.`
But reset states: `This feature is only possible if you have installed Helper Script.`

Chicken egg problem !
I guess we could install the script without reseting if need to get the reset script...

REM: Still using v3.7 and figuring out the least painful way to eventually upgrade.

**Comment by Guilouz** [comment]
I have updated it.
